packages/runtime/test-runtime-utils/src/test/mockLoggerErrorType.spec.ts

```diff
@@ -0,0 +1,49 @@
+/*!
```

Comment:
I wouldn't put "mockLogger" in the file name, this isn't testing the mock logger.  It's using it.  And double-check that this is the right package for this test.  It should go close to the rest of the error logging code.